openapi/components/examples/lambdas/200-realms.json

```diff
@@ -0,0 +1,57 @@
+[
+    {
+        "serverName": "loki",
+        "url": "https://interconnected.online",
+        "layer": "amber",
```

Comment:
We should remove layers from this response, as they won't be present any more

Reply:
done